docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+shipit:
+  build: ./
+  command: python kickoff-web.py -d sqlite:///update.db --username=admin --password=password --host=0.0.0.0 --port=5000 --verbose
```

Comment:
Suspected that issue may have been because I have mysql-server installed locally and running (I use it for my relengapi work) -- so I stopped that, and then I get:
```
shipit_1   | sqlalchemy.exc.OperationalError: (OperationalError) (1130, "Host '172.17.0.52' is not allowed to connect to this MySQL server") None None
```

as my exception